### PR TITLE
Add PR template

### PR DESCRIPTION
**Pull request by VitikaSoni**
Signed-off-by: VitikaSoni <vitika.program@gmail.com>

### CODE REVIEW

**Review by kingthorin — APPROVED** [review]

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+## Overview
+Briefly describe the purpose, goals, and changes or improvements made in this pull request.
+
+## Related Issues
+Specify any related issues or pull requests by linking to them.
+
+## Checklist
+- [ ] Update help
+- [ ] Update changelog
+- [ ] Use `labelFor` (if applicable) - [Issue #943](https://github.com/zaproxy/zaproxy/issues/943)
```
Comment (ricekot) [inline .github/pull_request_template.md:10]: Will this cause all PRs to link back to this issue?

Comment (kingthorin) [reply .github/pull_request_template.md:10]: Hmmm probably. I’ll see if I can come up with an Oracle or OpenJDK reference.

Comment (VitikaSoni) [reply .github/pull_request_template.md:10]: [This](https://docs.oracle.com/javase/8/docs/api/javax/swing/JLabel.html#setLabelFor-java.awt.Component-)?

Comment (kingthorin) [reply .github/pull_request_template.md:10]: Yup that seems good.

Comment (thc202) [reply .github/pull_request_template.md:10]: IMO this should be part of the dev guidelines.

Comment (kingthorin) [reply .github/pull_request_template.md:10]: Okay, I’d be happy to see this bullet handled that way too 🙂

**Review by kingthorin — APPROVED** [review]

**Review by psiinon — APPROVED** [review]

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**